File: PYTHON/EJERCICIOS/CONDICIONALES/Ejercicio03Divisor-cero.py
"""
Escribir un programa que pida al usuario 
dos números y muestre por pantalla su 
división. Si el divisor es cero el 
programa debe mostrar un error.
De lo contrario mostrar la división.
"""
    
#variables
numero_01 = int(input("Ingresa un numero: "))
divisor = int(input("Ingresa el divisor: "))

#validamos si es cero el divisor
if divisor == 0:
    print("No se puede dividir por cero.. ERRORR!!!")
else:
    #dividimos y mostramos la division
    division = int(numero_01 / divisor);
    print("La division es: ",division)    

File: JAVA/EJERCICIOS/CONDICIONALES/Ejercicio03DivisorCero.java
package JAVA.EJERCICIOS.CONDICIONALES;

import java.util.Scanner;
public class Ejercicio03DivisorCero {
    static Scanner leer = new Scanner(System.in);
    public static void main(String[] args) {

        /*variables */
        int numero, divisor;
        /*pedimos al usuario los datos */
        System.out.print("Ingrese un numero: ");
        numero = leer.nextInt();
        System.out.print("Ingrese el numero divisor: ");
        divisor = leer.nextInt();

        /*validamos mediante condicionales */
        if (divisor == 0) {
            System.out.println("ERROR!!!, no se puede dividir por cero...");
        } else {
            int division;
            division = numero / divisor;
            System.out.println("la division es: " + division);
        }
    }
}
